.travis.yml

```diff
@@ -14,7 +14,7 @@ notifications:
 addons:
   apt:
     packages:
-    - oracle-java8-set-default
+    - openjdk-8-jre-headless
```

Comment:
That one was done by #1127 so this just needs a rebase I guess?

Reply:
Removed